charts/shoot-core/charts/cloud-controller-manager/templates/webhook-gardener-add-initializer.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: persistent-volume-binder-initializer
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  - initialize
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: persistent-volume-binder-initializer
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: persistent-volume-binder-initializer
+subjects:
+- kind: ServiceAccount
+  name: persistent-volume-binder
+  namespace: kube-system
+---
+apiVersion: {{ include "webhookadmissionregistration" . }}
+kind: MutatingWebhookConfiguration
+metadata:
+  name: add-initializers-only-for-cloud-pvs
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+webhooks:
+- name: add-initializers-only-for-cloud-pvs.gardener.cloud
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    resources:
+    - persistentvolumes
+  failurePolicy: Fail
+  clientConfig:
+    url: https://gardener-external-admission-controller.garden/webhooks/mutate-pv-initializers
```

Comment:
Better to use a [service](https://github.com/kubernetes/api/blob/644117120187a38be1542f4598cbcf7a1b19a015/admissionregistration/v1beta1/types.go#L285) reference as DNS might be down.

Reply:
Service would mean that the admission plugin must run in the Shoot cluster.